libcaja-private/caja-undostack-manager.c

```diff
@@ -220,11 +220,6 @@ caja_undostack_manager_class_init (CajaUndoStackManagerClass * klass)
       "Number of undo levels to be stored",
       1, UINT_MAX, 30, G_PARAM_READWRITE | G_PARAM_CONSTRUCT);
 
-  confirm_delete =
```

Comment:
`confirm_delete ` not cleanly removed
```diff
--- a/libcaja-private/caja-undostack-manager.c
+++ b/libcaja-private/caja-undostack-manager.c
@@ -109,7 +109,7 @@ struct _CajaUndoStackManagerPrivate
  ***************************************************************** */
 enum
 {
-  PROP_UNDOSTACK_MANAGER_0, PROP_UNDO_LEVELS, PROP_CONFIRM_DELETE
+  PROP_UNDOSTACK_MANAGER_0, PROP_UNDO_LEVELS
 };
 
 static void caja_undostack_manager_set_property (GObject * object,
@@ -212,7 +212,6 @@ static void
 caja_undostack_manager_class_init (CajaUndoStackManagerClass * klass)
 {
   GParamSpec *undo_levels;
-  GParamSpec *confirm_delete;
   GObjectClass *g_object_class;
 
   /* Create properties */
@@ -230,9 +229,6 @@ caja_undostack_manager_class_init (CajaUndoStackManagerClass * klass)
   g_object_class_install_property (g_object_class, PROP_UNDO_LEVELS,
       undo_levels);
 
-  g_object_class_install_property (g_object_class, PROP_CONFIRM_DELETE,
-      confirm_delete);
-
   /* The UI menu needs to update its status */
   g_signal_new ("request-menu-update",
       G_TYPE_FROM_CLASS (klass),
```

Reply:
Apparently my search for these was incomplete, maybe a string search problem, maybe just working too fast. Anyway, took those instances out , rebuilt locally, and it worked. Again (and as expected) the warnings in a terminal from which sudo caja is involved do not appear. Force-pushed your change. Thanks for catching that.